tldw_Server_API/tests/Embeddings/test_embeddings_endpoint_cache_identity.py

```
@@ -0,0 +1,84 @@
+        return self.data.get(key)
+
+    async def set(self, key, value):
+        self.set_keys.append(key)
+        self.data[key] = value
+
+
+def test_local_api_backend_identity_strips_credentials_and_query():
+    import tldw_Server_API.app.api.v1.endpoints.embeddings_v5_production_enhanced as mod
+
+    clean_url = "http://backend-one.example:8080/v1"
+    secret_url = "http://user:pass@backend-one.example:8080/v1?token=abc#fragment"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "secret_url"

(S105)